hudi-client/hudi-client-common/src/main/java/org/apache/hudi/table/marker/DirectWriteMarkers.java

```diff
@@ -135,18 +152,36 @@ private String translateMarkerToDataPath(String markerPath) {
     return stripMarkerSuffix(rPath);
   }
 
+  public static String stripMarkerSuffix(String path) {
+    return path.substring(0, path.indexOf(HoodieTableMetaClient.INPROGRESS_MARKER_EXTN));
+  }
+
+  public static String stripOldStyleMarkerSuffix(String path) {
```

Comment:
if we change marker file naming, we should see if we can introduce versioning for those? (like we do with other places where bits are written to storage?)